repository/confluent-schema-registry/docs/v1.0/security.md

```diff
@@ -0,0 +1,103 @@
+# Security
+
+The Confluent Schema Registry service supports native transport **encryption**, **authentication**, and **authorization** mechanisms to connect with Kafka’s brokers configured for secured connections. The service provides automation and orchestration to simplify the use of these important features. For more information on Confluent Schema Registry’s security while connecting with Kafka Brokers over secured connections, read the [security](https://docs.confluent.io/current/kafka/encryption.html#sr) section of official Confluent Schema Registry documentation.
```

Comment:
```suggestion
The KUDO Confluent Schema Registry service supports Confluent Schema Registry's native transport **encryption** mechanism to connect with Kafka’s brokers configured for secured connections. The service provides automation and orchestration to simplify the use of these important features. For more information on Confluent Schema Registry’s security, read the [security](https://docs.confluent.io/current/schema-registry/security/index.html) section of official Confluent Schema Registry documentation.
```

Reply:
@shubhanilBag This sentence `while connecting with Kafka Brokers over secured connections` is crucial since we're only targeting the connectivity of SR with Kafka brokers over SSL (And thus documentation is about that only, otherwise link will be https://docs.confluent.io/current/schema-registry/security/index.html)

Reply:
It under a general `#Security` header, plus you are already mentioning that in the first sentence. It 's better to provide the official **Schema Registry** docs that contain all details rather than providing only a link from Confluent Kafka's doc